doc-botonic/docs/welcome.md

```diff
@@ -0,0 +1,41 @@
+---
+id: welcome
+title: Welcome to Botonic User Guide
+sidebar_label: Welcome to Botonic User Guide  
+---
+
+> ## Chatbot vs Conversational App
```

Comment:
I see the use of `>` in titles everywhere. This is not good for 2 reasons:
1) The UI that displays looks like a "quote" or an "alert", and that's the intended use. It makes the title look "less title".
2) It generates the HTML inside a `<blockquote>` tag, this will confuse Google crawlers and hurt our SEO.

If you want to change the general styles of titles, subtitles, etc, I'm sure there's a way to do it with custom CSS

Reply:
Block removed.